src/gt4py/backend/dace_backend.py

```diff
@@ -50,6 +49,13 @@
     from gt4py.stencil_object import StencilObject
 
 
+def _serialize_sdfg(sdfg: dace.SDFG):
+    for sd in sdfg.all_sdfgs_recursive():
+        sd.transformation_hist = []
+        sd.orig_sdfg = None
+    return dumps(sdfg)
```

Comment:
This is a bit unintuitive: `_serialize_sdfg` actually mutates the input.

Reply:
Done.